examples/simple_config.py

```diff
@@ -27,7 +27,7 @@
     "hosts": [{"host": "localhost", "port": 9200}],
     "http_compress": True,
     "http_auth": ("admin", "admin"),
-    "use_ssl": False,
+    "use_ssl": True,
```

Comment:
Looks like there are a few more of these hidden in the documentation.
`grep -R use_ssl.*False .` finds 4